docs/workforce-management-framework/topics/ref-introducing-raincatcher.adoc

```diff
@@ -0,0 +1,70 @@
+[id='ref-introducing-raincatcher-{chapter}']
+= {WFM-RC-NameShort} Reference Material
+
+This section includes:
+
+. xref:highlighting-raincatcher-areas-of-focus-{chapter}[Highlighting {WFM-RC-NameShort} Areas of Focus]
+. xref:describing-raincatcher-technologies-and-technicalities-{chapter}[Describing {WFM-RC-NameShort} Technologies and Technicalities]
+. xref:identifying-raincatcher-repositories-{chapter}[Identifying {WFM-RC-NameShort} Repositories]
+. xref:highlighting-raincatcher-tech-preview-differences-{chapter}[Highlighting {WFM-RC-NameShort} and 'Tech Preview' Differences]
+
+[id='highlighting-raincatcher-areas-of-focus-{chapter}']
+== Highlighting {WFM-RC-NameShort} Areas of Focus
+
+.Areas of Focus
+|===
+|Area |Focus
+
+|Framework
+|A set of modules for building mobile and website Workforce Management solutions
+
+|Modules
+|Modules specific to {WFM-RC-NameShort}, for example: signatures, scheduler
+
+|Integrations
+|Integrations provide, for example: authentication, storage, offline capabilities, server side synchronization
+
+|User Interfaces
+|Graphical User Interfaces are implemented in AngularJS and Angular
+|===
+
+[id='describing-raincatcher-technologies-and-technicalities-{chapter}']
+== Describing {WFM-RC-NameShort} Technologies and Technicalities
+
+* link:https://nodejs.org/en/[Node.js]
+* Hybrid mobile development using link:https://www.typescriptlang.org/docs/home.html[TypeScript](cordova/web/angular)
+* Object Oriented approach (classes and interfaces)
+* Focus on module reuse and flexibility
```

Comment:
If we _briefly_ explain "how", I'm in favour of keeping this line of code, otherwise it can be taken out as it doesn't really do anything for the reader